src/test/java/uk/ac/ebi/ena/webin/cli/manifest/ManifestReaderJsonTest.java

```diff
@@ -148,6 +159,23 @@ public class ManifestReaderJsonTest {
                 .findFirst().isPresent());
     }
 
+    @Test public void testSampleJsonValue() throws JsonProcessingException {
+        TestManifestReader manifestReader = new TestManifestReader(new ManifestFieldDefinition.Builder()
+                .meta().required().name("sample").desc("some desc")
+                .build());
+
+        File manifestFile = new File(WebinCliTestUtils.getResourceDir("uk/ac/ebi/ena/webin/cli/manifests/json/valid/"),
+                "valid-sample.json");
+
+        manifestReader.readManifest(Paths.get("."), manifestFile);
+
+        ManifestReaderResult readerResult = manifestReader.getManifestReaderResult();
+
+        ManifestFieldValue sampleFieldValue = readerResult.getFields().stream()
+                .filter(field -> field.getName().equals("sample")).findFirst().get();
+        JsonNode sampleNode = new ObjectMapper().readTree(sampleFieldValue.getValue());
```

Comment:
I wonder if we should assert the whole sample JSON string or the sample JSON object here instead.